packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1014,7 +1014,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     private readonly createContainerMetadata: ICreateContainerMetadata;
-    private summaryCount: number | undefined;
+    // The summary number of the next summary that will be generated for this container. This is incremented every time
+    // a summary is generated.
+    private nextSummaryNumber: number;
```

Comment:
nit: I think this should be a doc comment since it's attached to a specific member variable

Reply:
You just mean they should be within /** */ right?

Reply:
Yes, not sure what the exact naming for those comments is

Reply:
Me neither. That's why I thought I will confirm :-). But good point, that's how these comments should be.